ideas/314-tribute-to-talk.md

```diff
@@ -0,0 +1,69 @@
+TtT Swarm
```

Comment:
Missing metadata

Reply:
Fixed. Spoke with Nabil and he'll handle budget post research phase.